sources/github.js

```diff
@@ -1,41 +1,97 @@
+/**
+*  Wrapper for GitHub API.
+*/
+
 var request = require('request');
 var config = require('../config.js');
+var rp = require('request-promise');
+
+var ENDPOINT = 'https://api.github.com/repos/';
 
-function createOptions(repo) {
+// Options when using GitHub API. Ex. Commits
+function createOptions(owner, repo) {
   return {
-    url: 'https://api.github.com/repos/' + repo + '/commits',
+    url: ENDPOINT + owner + '/'+ repo + '/commits',
     headers: {
       'Authorization': 'token ' + config.githubKey, // Bens access token
       'user-agent': 'Klipfolio-Decode-2016-Fall'
     }
   };
-}
+};
+
+/**
+* @desc Make calls to the GitHub API to retrieve data.
+*
+* @param measurement String - The measurement we want to store into the database
+* @param owner String - The user that owns a GitHub repo.
+* @param repo String - The name of the repo we want to get dataform.
+*
+* @returns request promise
+*/
+module.exports.queryGithubData = function(measurement, owner, repo){
```

Comment:
So this function would have to change as well to reflect how we are referring to repos as `author/name` instead of separated.